package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "next dev",
     "debug": "NODE_OPTIONS='--inspect' next dev",
     "build": "next build",
+    "lint": "npx eslint . --ext .js,.jsx",
```

Comment:
why not use [next lint](https://nextjs.org/docs/basic-features/eslint)?

Reply:
I was unaware of that, but I'll check it out.

Reply:
But what I did was just use the same command we're using in the automated CI check.